src/ocean/text/Util.d

```diff
@@ -785,6 +788,136 @@ bool isSpace(T) (T c)
            return (c <= 32 && (c is ' ' || c is '\t' || c is '\r' || c is '\n' || c is '\f' || c is '\v')) || (c is '\u2028' || c is '\u2029');
 }
 
+/*******************************************************************************
+
+    Enumeration of the different ways in which a string can be padded in order
+    to increase its length.
+
+*******************************************************************************/
+
+public enum PadType : ubyte
+{
+    PadPre,    // Padding is only applied before the string
+    PadPost,   // Padding is only applied after the string
+    PadAround  // Padding is applied on both sides of the string
+}
+
+/*******************************************************************************
+
+    Shortens/Lengthens a string as necessary to get it to a desired length.
```

Comment:
I think, in a library, it's always nicer to have smaller, building blocks. We can have a helper/wrapper, too, that combines them in the way which you envisage using them.